/limit_switch_node:
  ros__parameters:
    delta_time: 5.0
    topic_name: "oggetto_arrivato"
